{ 
  "text" : "NULL pointer dereference in Google TensorFlow before 1.12.2 could cause a denial of service via an invalid GIF file.\n" ,
  "entities" : [ { 
    "eid" : "E1" ,
    "keyid" : "Identity#c822c1b63853ed273b89687ac505f9fa" ,
    "type" : "Identity" ,
    "subType" : "Organization" ,
    "value" : "google" ,
    "begin" : 28 ,
    "end" : 34 ,
    "canonicalForm" : "Google" ,
    "score" : "0.0"
  }
  , { 
    "eid" : "E2" ,
    "keyid" : "TTP#9c98ef6a7451d6f82ca2194ee23d1acb" ,
    "type" : "TTP" ,
    "subType" : "TTP" ,
    "value" : "denial of service" ,
    "begin" : 74 ,
    "end" : 91 ,
    "canonicalForm" : "denial of service" ,
    "score" : "0.0"
  }
  , { 
    "eid" : "E3" ,
    "keyid" : "8f1bb270a49d8c3ed3be5db23adb9a7b" ,
    "type" : "MiscEntity" ,
    "subType" : "file" ,
    "value" : "invalid GIF file" ,
    "begin" : 99 ,
    "end" : 115 ,
    "canonicalForm" : "invalid GIF file" ,
    "score" : "0.0"
  }
  ,  ] ,
  "relations" : [ { 
    "rid" : "R1" ,
    "type" : "cause_via" ,
    "predicate" : { 
      "begin" : 60 ,
      "end" : 95 ,
      "surfaceForm" : "could cause a denial of service via"
    }
     ,
    "arguments" : [ { 
      "argnum" : 1 ,
      "begin" : 74 ,
      "end" : 91 ,
      "eid" : "E2" ,
      "canonicalForm" : "denial of service"
    }
    , { 
      "argnum" : 2 ,
      "begin" : 99 ,
      "end" : 115 ,
      "eid" : "E3" ,
      "canonicalForm" : "invalid GIF file"
    }
    ,  ]
  }
  , { 
    "rid" : "R2" ,
    "type" : "cause" ,
    "predicate" : { 
      "begin" : 60 ,
      "end" : 71 ,
      "surfaceForm" : "could cause"
    }
     ,
    "arguments" : [ { 
      "argnum" : 1 ,
      "begin" : 28 ,
      "end" : 34 ,
      "eid" : "E1" ,
      "canonicalForm" : "Google"
    }
    , { 
      "argnum" : 2 ,
      "begin" : 74 ,
      "end" : 91 ,
      "eid" : "E2" ,
      "canonicalForm" : "denial of service"
    }
    ,  ]
  }
  ,  ]
}
